### ISSUE TITLE
Become a new maintainer

### ISSUE THREAD

**Issue by VadymMatus** [niklauslee/staruml-python#11]
I really need reverse engineering from Python code, I don't see the point of starting a new project, I want to add this feature here, besides, there is already an issue on this function, can I somehow help?

**Comment by niklauslee** [comment]
@VadymMatus Thank you for your interest for contribution. Please write your own code and then send us a PR (Pull Request). If you have interest to be a new maintainer of this repository, I'd like to add you as a maintainer after the PR.

@IvanMtze Do you have any progress on the issue #3?

**Comment by IvanMtze** [comment]
@niklauslee By this moment I don't have any progress on that issue. However if @VadymMatus has done something feel free create a PR , then I can help to finish the issue.

**Comment by VadymMatus** [comment]
Yes, I'm working on it, now I'm solving the problem with parsing Python source codes, I was looking for a JS solution for parsing Python AST, but I didn't find anything worthwhile, other extensions for StarUML for reverse engineering use the jison tool, now it's worth a choice, set up grammar for jison to get a parser in JS or call a small Python script that will return an AST tree in JSON format that can already be converted to UML. In the first case, it will take a lot of time, and there will be a problem with the constant addition of new features in Python, in the second case there will be a constantly maintained module (ast), but this is an external dependency (from the system) for extension ...
What do you think about this?

**Comment by IvanMtze** [comment]
HI @VadymMatus  I guess we should go for the first. I guess the addition of new features in Python is not a big problem that the one that is gonna be constantly maintained. However, I'm not really sure if there is a reason to prefer jison over other tools.  I will create a new branch so I can help you with the implementation to get this done as early as possible. @niklauslee Do you have any reason to avoid using jison?

**Comment by niklauslee** [comment]
@VadymMatus @IvanMtze It would be nice if it was developed purely in JavaScript, but it doesn't matter if it isn't. :)
